astropy/io/fits/tests/test_core.py

```diff
@@ -363,8 +363,8 @@ def test_extension_name_case_sensitive(self):
         runtime works.
         """
 
-        if 'PYFITS_EXTENSION_NAME_CASE_SENSITIVE' in os.environ:
-            del os.environ['PYFITS_EXTENSION_NAME_CASE_SENSITIVE']
+        if 'IO_FITS_EXTENSION_NAME_CASE_SENSITIVE' in os.environ:
+            del os.environ['IO_FITS_EXTENSION_NAME_CASE_SENSITIVE']
```

Comment:
It seems that this environment variable was replaced by a config item `conf.extension_name_case_sensitive` so these 2 lines can be removed.